packages/graphql/src/graphql/tree.js

```diff
@@ -0,0 +1,45 @@
+const uniqBy = require("lodash/uniqBy");
```

Comment:
Why are all these files not typescripted? 😝

Reply:
We don't have typescript support on node (yet). A bundler would create an overhead during development (without proper setup). Migrating to typescript is on the backlog though! 😃